source/common/tracing/zipkin/zipkin_core_types.h

```
@@ -302,7 +302,8 @@ class Span : public ZipkinBase {
    * Default constructor. Creates an empty span.
    */
   Span()
-      : trace_id_(0), name_(), id_(0), debug_(false), monotonic_start_time_(0), tracer_(nullptr) {}
+      : trace_id_(0), name_(), id_(0), debug_(false), sampled_(true), monotonic_start_time_(0),
```

again default to sampled false on dummy data